operator/hack/deploy-gcp-storage-secret.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Should use `env` to make it more portable?

```suggestion
#!/usr/bin/env bash
```